notebooks/how_to_pyscf.md

```diff
@@ -0,0 +1,117 @@
+# How to write PySCF output into TREXIO format
+
+This tutorial shows how to use PySCF (https://pyscf.org/) to save computational results in TREXIO format. 
+
+## Importing TREXIO
+
+TREXIO is avialble throught the pyscf-forge repository (https://github.com/pyscf/pyscf-forge). After installing PySCF-FORGE, you can access the TREXIO library within PySCF using the following import statement:
+
+```python
+from pyscf.tools import trexio
+```
+
+## How to register RHF results
+
+Once a molecule is defined, you can register the Restricted Hartree-Fock (RHF) results into TREXIO. If needed, it is also possible to register UHF wavefunction.
+In this tutorial, the HDF5 backend is used to store the results.
+
+```python
+# Perform RHF calculation
+mf = scf.RHF(mol)
+mf.kernel()
+
+# Save results in TREXIO format
+trexio_file = 'rhf.hdf5'
+to_trexio(mf, trexio_file)
```

Comment:
Please define where all your functions come from. If you try to run your Jupyter notebook as it is right now - the execution will fail because `to_trexio` and a bunch of other TREXIO-related functions will not be found. 

Since this is a tutorial for newcomers - we try to be more pedagogic and give more details so that the user always knows where the functions come from. In your case, for example, one can do:

```python
from pyscf.tools import trexio as pyscf_trexio
pyscf_trexio.to_trexio(...)
```